specs/derivation.md

```diff
@@ -294,7 +294,7 @@ Batcher transactions are encoded as `version_byte ++ rollup_payload` (where `++`
 | 0              | `frame ...` (one or more frames, concatenated) |
 
 Unknown versions make the batcher transaction invalid (it must be ignored by the rollup node).
-All frames in a batcher transaction must be parsable. If any one frame fails to parse, the all frames in the
+All frames in a batcher transaction must be parsable. If any one frame fails to parse, then all frames in the
```

Comment:
For the same words with Optimism. 
https://github.com/ethereum-optimism/optimism/blob/b4c39a90f996b7131a9d03c7937e31e38445f22d/specs/derivation.md?plain=1#L310
```suggestion
All frames in a batcher transaction must be parsable. If any one frame fails to parse, the all frames in the
```